src/pages/BookmarksPage.js

```diff
@@ -9,39 +11,48 @@ export default function BookmarksPage({
   handleChangeFilterBookmarks,
 }) {
   return (
-    <DrinklistContainer>
-      <FilterDrinks
-        handleChangeFilter={handleChangeFilterBookmarks}
-        currentFilter={currentFilterBookmarks}
-      />
-      {drinks &&
-        drinks
-          .filter(drink => drink.isBookmarked === true)
-          .filter(
-            drink =>
-              drink.strAlcoholic.toLowerCase() === currentFilterBookmarks ||
-              currentFilterBookmarks === 'all'
-          )
-          .map(drink => (
-            <DrinkCard
-              key={drink.idDrink}
-              drink={drink}
-              toggleBookmark={toggleBookmark}
-            />
-          ))}
-      {drinks.filter(drink => drink.isBookmarked === true).length < 1 && (
-        <EmptyState>
-          There is nothing on your favorites list at the moment. Start with
-          adding your Drinks by clicking on the sun icon.
-        </EmptyState>
-      )}
-    </DrinklistContainer>
+    <PageContainer>
+      <Header>Favorites</Header>
+      <DrinklistContainer>
+        <FilterDrinks
+          handleChangeFilter={handleChangeFilterBookmarks}
+          currentFilter={currentFilterBookmarks}
+        />
+        {drinks &&
+          drinks
+            .filter(drink => drink.isBookmarked === true)
+            .filter(
+              drink =>
+                drink.strAlcoholic.toLowerCase() === currentFilterBookmarks ||
+                currentFilterBookmarks === 'all'
+            )
+            .map(drink => (
+              <DrinkCard
+                key={drink.idDrink}
+                drink={drink}
+                toggleBookmark={toggleBookmark}
+              />
+            ))}
+        {drinks.filter(drink => drink.isBookmarked === true).length < 1 && (
```

Comment:
You are using `drinks.filter(drink => drink.isBookmarked === true` the second time on this page; might it be a better (readable) solution to create a variable with this array at the beginning and use it here and in line 22ff. above?

Reply:
Thanks for this . I will take this for the refactoring user story that I will start the next days. There is a lot to optimize I think...